src/main/java/seedu/address/logic/parser/HelpCommandParser.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.AddCommand;
+import seedu.address.logic.commands.ClearCommand;
+import seedu.address.logic.commands.DeleteCommand;
+import seedu.address.logic.commands.ExitCommand;
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.commands.HelpCommand;
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.commands.edit.EditCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new HelpCommand object
+ */
+public class HelpCommandParser implements Parser<HelpCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the HelpCommand
+     * and returns a HelpCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    @Override
+    public HelpCommand parse(String keyword) throws ParseException {
+        String trimmedLowercaseKeyword = keyword.trim().toLowerCase();
+        if (!trimmedLowercaseKeyword.isEmpty()
+            && !trimmedLowercaseKeyword.equals(AddCommand.COMMAND_WORD)
```

Comment:
maybe can add `&& !trimmedLowercaseKeyword.equals(HelpCommand.COMMAND_WORD)` to allow `help help` to be a valid command

Reply:
Very good spot! Thank you👍 @weiseng18 I just realized that I leave this thing out and caused `help help` command to fail, the test cases didn't catch this error either so I have fixed them also. The error message for `ParseException` is also wrong(I didn't use the `INVALID_HELP_KEYWORD` message I come up with lol), but now everything should be fine. Thank you!